python/lsst/meas/base/plugins.py

```diff
@@ -400,6 +400,134 @@ def fail(self, measRecord, error=None):
 """
 
 
+class EvaluateLocalPhotoCalibPluginConfig(BaseMeasurementPluginConfig):
+    """Configuration for the variance calculation plugin.
+    """
+    pass
+
+
+class EvaluateLocalPhotoCalibPlugin(GenericPlugin):
+    """Evaluate the local value of the Photometric Calibration in the exposure.
+
+    The aim is to store the local calib value within the catalog for later
+    use in the Science Data Model functors.
+    """
+
+    ConfigClass = VarianceConfig
+
+    FAILURE_BAD_CENTROID = 1
+    """Denotes failures due to bad centroiding (`int`).
+    """
+
+    FAILURE_EMPTY_FOOTPRINT = 2
+    """Denotes failures due to a lack of usable pixels (`int`).
+    """
+
+    @classmethod
+    def getExecutionOrder(cls):
+        return BasePlugin.FLUX_ORDER
+
+    def __init__(self, config, name, schema, metadata):
+        GenericPlugin.__init__(self, config, name, schema, metadata)
+        self.photoKey = schema.addField(
+            "base_LocalPhotoCalib",
```

Comment:
Use `name` to reuse the name you specify when calling makeSingleFramePlugin Below.

Reply:
So you're saying that in the class `__init__` I should use the `name` variable in to replace explicitly writing `base_LocalPhotoCalib` or `base_LocalWcs`?